### DISCUSSION TITLE
Clicking on Birdseye Frame Idea

### DISCUSSION THREAD

**Discussion by psCLE** [blakeblackshear/frigate#15142] [Ideas]
Clicking on an active Birdseye frame would go to the full resolution live feed.

**Comment by ElectronicBattle** [comment]
I've just got my nine cameras working perfectly and use of birdseye alongside several camera groups means my UI is now suitable for a desktop monitor, for one camera instead of all, for use on a tablet.

It would be a great option to be able to "click into" the birdseye grid and select one of the feeds there to become full screen, even if it is "only" the substream (which might be 640x480 rather than its main HD equivalent).

I can imagine it being very effective  for a touchscreen : touch one = full screen, touch two = reverts to birdseye grid.

On a desktop, a double-click in and another back out would work: even a double-click on the top half of a grid's subframe would switch to fullscreen SD, and the lower half would be HD etc.
